test/langtools/tools/doclint/AnchorTest.java

```diff
@@ -1,6 +1,6 @@
 /*
  * @test /nodynamiccopyright/
- * @bug 8004832
+ * @bug 8004832 8247957
```

Comment:
The bug number should only be added when the test can be considered to be a significant test of the work.

In general, don't bother to add the bug number when a test is just incidentally affected by the work.

In this case, there appears to be no other change to the file, but I guess OK, you've changed to golden output file.

Reply:
Understood.  Thanks for your instruction.